common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,318 @@
+"""
+Acceptance Tests for Course Information
+"""
+import uuid
+
+from bok_choy.web_app_test import WebAppTest
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+from flaky import flaky
+
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+from ...pages.studio.overview import CourseOutlinePage
+
+
+def _create_course(self):
```

Comment:
A better pattern would be to create a mixin or a base class that the test classes can extend from.